cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -225,6 +225,22 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
         return lst;
     };
 
+    auto adapt_write_dfs = [](std::vector<std::variant<VersionedItem, DataError>> && ret) -> py::list {
```

Comment:
Is this needed? `pybind11` will convert a vector to a list anyway, and I think it understands `std::variant` so this behaviour should come for free?
If it is necessary, `adapt_write_dfs` does not describe what this function does at all.